React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -591,14 +591,27 @@ - (void)invalidateLayer
 
     RCTBorderColors borderColors = RCTCreateRCTBorderColorsFromBorderColors(borderMetrics.borderColors);
 
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
```

Comment:
Doing it this way means we'd implement Fabric the same way we implemented paper: with lots of ifdefs. We can talk about this more in the future, but I'm guessing you did your own personal investigation and concluded that makes sense?

Reply:
It is not ideal - but RCTGetBorderImage is also a C function and doesn't take default argument values.
This change allows to at least compile and run Fabric for iOS with this repro so I think it will be easier to work on the 'actual' changes needed to make macOS working.

But we can also keep this one open for a while.
I think once we have a strategy for 'how' to enable Fabric we can still think of making a series of smaller contributions instead of 1 gigantic change which is hard to land (or revert?)